source/configuration_manual/event_filter.rst

```diff
@@ -93,14 +93,18 @@ A more complicated example::
     NOT category=debug AND NOT (bytes_in<1024 OR bytes_out<1024))
 
 .. versionadded:: v2.4.0;v3.0.0 Sizes can be expressed using the unit values
-   ``B`` - which represents single byte values - as well as ``K``, ``M``, ``G``
-   and ``T`` which are all powers of 1024. If no unit is specified ``B`` is
-   used by default.
+   ``B`` - which represents single byte values - as well as ``KB``, ``MB``,
+   ``GB`` and ``TB`` which are all powers of 1024. If no unit is specified
+   ``B`` is used by default. All size units can be lower case as well.
```

Comment:
I think not just lowercase, but mixed case also. So I'd say "All size units are case-insensitive."